utils/remote-run

```diff
@@ -61,14 +61,14 @@ class CommandRunner(object):
             subprocess.check_call(mkdir_command)
 
         # Fetch the remote files.
-        sftp_commands = ("-get {0} {1}".format(quote(remote_file), 
+        sftp_commands = ("-get {0} {1}".format(quote(remote_file),
                                                quote(local_file))
                          for local_file, remote_file
                          in local_to_remote_files.viewitems())
         self.run_sftp(sftp_commands)
 
     def run_remote(self, command, remote_env={}):
-        env_strings = ['{0}={1}'.format(k,v) for k,v in remote_env.viewitems()]
+        env_strings = ['{0}={1}'.format(k,v) for k,v in remote_env.items()]
```

Comment:
Besides this `viewitems`, there's a `viewvalues` in L44, another `viewitems` in L51 and L67, a `viewkeys` in L56. Didn’t you have problems with those in Python 3?

Reply:
I don't know that I ever finished getting this particular script working 😅